kaspad/src/args.rs

```diff
@@ -111,23 +117,37 @@ pub fn cli(defaults: &Defaults) -> Command {
                 .num_args(0..=1)
                 .require_equals(true)
                 .default_missing_value(defaults.rpclisten_json)
+                .value_parser(clap::value_parser!(ContextualNetAddress))
                 .help(format!("Interface:port to listen for wRPC JSON connections (default: {}).", defaults.rpclisten_json)),
         )
         .arg(arg!(--unsaferpc "Enable RPC commands which affect the state of the node"))
         .arg(
-            Arg::new("connect")
+            Arg::new("connect-peers")
                 .long("connect")
                 .value_name("IP[:PORT]")
-                .num_args(0..=1)
+                .action(ArgAction::Append)
+                .num_args(0..=10) // TODO: define an upper bound
```

Comment:
Are you sure `num_args` is relevant when using `ArgAction::Append` ? we should manually test it.

Reply:
In general I'm not sure I understand the role of `.num_args(0..=1)` in all other flags

Reply:
I agree on both remarks.

I removed all `.num_args(iter)` and tested manually. The system exhibits the desired behavior as expected.